src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
You probably need to add a transition name here too. You may think that it's obvious because you have just one transition here, but mentors recommend adding the name anyway. So` transition-duration: 300ms` should become `transition: color 300ms;` in my opinion.